[SSH] -sh: /dev/fd/64: line 14: syntax error: unexpected "(" (expecting "}")

### Describe the issue you are experiencing

Since upgrading to Terminal & SSH 9.20.0 I see the following error when SSHing into the host:
```
System is ready! Use browser or app to configure.
-sh: /dev/fd/64: line 14: syntax error: unexpected "(" (expecting "}")
```

Amongst other things, this prevents VSCode Remote Extensions from working (local VSCode, not the add-on).


### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

Terminal & SSH

### What is the version of the add-on?

9.20.0

### Steps to reproduce the issue

The error is because the `root` user shell is set to `/bin/sh`, which is `/bin/busybox` not `/bin/bash`, while `/etc/profile.d/homeassistant.sh` contains the following:
```
# shellcheck disable=SC1090
source <(ha completion bash)
```

ie. it tries to run bash-specific shell syntax in non-bash shells.

Changing the root shell to `/bin/bash` fixed it for me

### System Health information

## System Information

version | core-2025.9.1
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
container_arch | aarch64
user | root
virtualenv | false
python_version | 3.13.7
os_name | Linux
os_version | 6.12.43-haos
arch | aarch64
timezone | America/Los_Angeles
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
HACS Data | ok
GitHub API Calls Remaining | 5000
Installed Version | 2.0.5
Stage | running
Available Repositories | 2202
Downloaded Repositories | 7

</details>

<details><summary>AccuWeather</summary>

can_reach_server | ok
-- | --
remaining_requests | 248

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | April 17, 2026 at 5:00 PM
relayer_connected | true
relayer_region | us-east-1
remote_enabled | true
remote_connected | true
alexa_enabled | true
google_enabled | true
cloud_ice_servers_enabled | true
remote_server | us-east-1-5.ui.nabu.casa
certificate_status | ready
instance_id | 858270d6d24b4b50887b97e8b326ad35
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 16.2
-- | --
update_channel | stable
supervisor_version | supervisor-2025.09.0
agent_version | 1.7.2
docker_version | 28.3.3
disk_total | 125.3 GB
disk_used | 18.4 GB
nameservers | 192.168.7.1, fdd8:fa25:dffa:1::1
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | 
board | generic-aarch64
supervisor_api | ok
version_api | ok
installed_addons | Matter Server (8.1.0), Studio Code Server (5.19.3), OpenThread Border Router (2.13.0), Mosquitto broker (6.5.2), Whisper (2.6.0), Piper (1.6.4), Music Assistant Server (2.5.8), Terminal & SSH (9.20.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 0
views | 10
mode | yaml

</details>

<details><summary>Network Configuration</summary>

adapters | lo (disabled), enp0s8 (enabled, default, auto), docker0 (disabled), hassio (disabled), veth7a3144d (disabled), veth1960570 (disabled), veth7d8c968 (disabled), vethf10419f (disabled), veth9b4b676 (disabled), veth67ffe2a (disabled), veth0bd4dee (disabled), vethee77270 (disabled), veth69fc3bc (disabled), wpan0 (disabled), vethe892a88 (disabled), vethb44a74c (disabled)
-- | --
ipv4_addresses | lo (127.0.0.1/8), enp0s8 (192.168.7.203/24), docker0 (172.30.232.1/23), hassio (172.30.32.1/23), veth7a3144d (), veth1960570 (), veth7d8c968 (), vethf10419f (), veth9b4b676 (), veth67ffe2a (), veth0bd4dee (), vethee77270 (), veth69fc3bc (), wpan0 (), vethe892a88 (), vethb44a74c ()
ipv6_addresses | lo (::1/128), enp0s8 (fdd8:fa25:dffa:1:df34:910:fcc5:8f4b/64, fe80::b607:4063:5d4b:5a20/64), docker0 (fe80::f4c8:b7ff:fe05:1199/64), hassio (fd0c:ac1e:2100::1/48, fe80::4c26:10ff:fe82:bad2/64), veth7a3144d (fe80::a4bd:1aff:fe64:3142/64), veth1960570 (fe80::a8e2:31ff:fee4:1a5/64), veth7d8c968 (fe80::c8b1:f3ff:fe63:4f73/64), vethf10419f (fe80::8cb3:dfff:fe22:cbfc/64), veth9b4b676 (fe80::9825:afff:feab:3bb9/64), veth67ffe2a (fe80::9844:41ff:fe43:5038/64), veth0bd4dee (fe80::485:35ff:fede:583d/64), vethee77270 (fe80::345f:f0ff:fef7:db03/64), veth69fc3bc (fe80::cc29:6fff:fec5:b266/64), wpan0 (fd16:15:12a0:0:4d38:68f5:8fc6:254e/64, fddb:a0f5:4dcf::ff:fe00:5400/64, fddb:a0f5:4dcf:0:b7a0:f6d2:6d1f:3bdc/64, fe80::a83a:a434:f40f:f942/64), vethe892a88 (fe80::602e:f9ff:fe05:38dc/64), vethb44a74c (fe80::c431:98ff:fe37:d3d2/64)
announce_addresses | 192.168.7.203, fdd8:fa25:dffa:1:df34:910:fcc5:8f4b, fe80::b607:4063:5d4b:5a20

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | August 16, 2025 at 7:37 PM
-- | --
current_recorder_run | September 10, 2025 at 11:06 PM
estimated_db_size | 163.54 MiB
database_engine | sqlite
database_version | 3.48.0

</details>

### Anything in the Supervisor logs that might be useful for us?

```txt

```

### Anything in the add-on logs that might be useful for us?

```txt

```

### Additional information

_No response_

--- a/ssh/CHANGELOG.md
+++ b/ssh/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.20.1
+
+- Fix default shell
+
 ## 9.20.0
 
 - Upgrade Home Assistant CLI to 4.41.0

--- a/ssh/Dockerfile
+++ b/ssh/Dockerfile
@@ -30,7 +30,7 @@ RUN \
         openssl-dev \
         zlib-dev \
     \
-    && sed -i "s/ash/bash/" /etc/passwd \
+    && sed -i "1s/sh/bash/" /etc/passwd \
     \
     && git clone --branch "v${LIBWEBSOCKETS_VERSION}" --depth=1 \
         https://github.com/warmcat/libwebsockets.git /tmp/libwebsockets \

--- a/ssh/Dockerfile
+++ b/ssh/Dockerfile
@@ -30,7 +30,7 @@ RUN \
         openssl-dev \
         zlib-dev \
     \
-    && sed -i "1s/sh/bash/" /etc/passwd \
+    && sed -i "s|/bin/sh|/bin/bash|" /etc/passwd \
     \
     && git clone --branch "v${LIBWEBSOCKETS_VERSION}" --depth=1 \
         https://github.com/warmcat/libwebsockets.git /tmp/libwebsockets \

--- a/ssh/config.yaml
+++ b/ssh/config.yaml
@@ -1,5 +1,5 @@
 ---
-version: 9.20.0
+version: 9.20.1
 slug: ssh
 name: Terminal & SSH
 description: Allow logging in remotely to Home Assistant using SSH

